src/Spec2-Examples/SpApplicationWithToolbar.class.st

```diff
@@ -42,6 +34,15 @@ SpApplicationWithToolbar >> addItemTo: aGroup [
 	self build
 ]
 
+{ #category : #layout }
+SpApplicationWithToolbar >> defaultLayout [
+
+	^ SpBoxLayout newTopToBottom
+		add: menu withConstraints: [ :constraints | constraints height: self class toolbarHeight ];
```

Comment:
in fact... add:height: would be not correct here (`add: ... expand: false` would be)

Reply:
Ok, I'll do that